codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -190,6 +190,9 @@ spec:
   - name: [[ lower $port.Name ]]
     port: {{ $[[ $operatorVar ]].ports.[[ $port.Name ]] }}
   [[- end ]]
+  [[- if gt $[[ $operatorVar ]].statsPort 0 ]]
```

Comment:
This won't be available to the Operator since it's not described in the Operator spec. In this skv2 PR, it should be enough to just remove the AdminService. The statsPort changes will all happen in the GME PR.

Reply:
Gotcha, I thought that would need to be specified in the template being written out as well; is that just taken care of by the overrides?

Reply:
Yup! The ui-auth.yaml only exists in gloo-mesh-enterprise. The value it pulls in will be something like `glooMeshMgmtServer.statsPort`, which only needs to be defined in the gloo-mesh-enterprise values.yaml. We are essentially pulling this port completely out of the generated Operator deployment yamls.